docs/data/charts/bars/StackBars.tsx

```diff
@@ -17,8 +17,13 @@ export default function StackBars() {
       ])}
       xAxis={[{ scaleType: 'band', dataKey: 'year' }]}
       slotProps={{ legend: { hidden: true } }}
-      width={600}
-      height={350}
+      {...baseProps}
     />
   );
 }
+
+const baseProps = {
+  width: 600,
+  height: 350,
+  margin: { left: 70 },
```

Comment:
All the demo changes are due to the sized version of the chart not being wrapped in a flex div anymore. So parts of the axis labels or the flow layout were disrupted.

Reply:
What about having the flex all the time?

The DOM inconsistency between responsive and not responsive container is somewhat an issue because you sometime have a div and sometime a SVG as the root component. Here it become even more an issue with the same component having different DOM structure depending on the `width` `height` props (It will be hard to explain that to users 😁)

Reply:
🤔 yeah true, let me test

Reply:
This allowed me to revert the demo changes as well and had no side-effects that I can see 😅